pykafka/producer.py

```diff
@@ -167,7 +292,8 @@ def _get_partition_msgs(partition_id, req):
                     to_retry.extend(_get_partition_msgs(partition, req))
```

Comment:
I know these lines didn't change in this pullreq, but I think it's worth flagging up that these produce-errors (the ones that bear a `continue`) only cause a log warning, so they're sort of silently dropped.